charts/rancher-istio/rancher-istio/100.0.1+up1.10.4/values.yaml

```diff
@@ -5,16 +5,15 @@ forceInstall: false
 
 installer:
   repository: rancher/istio-installer
-  tag: 1.10.4-rancher3
+  tag: 1.10.4-rancher4
   ##releaseMirror are configurations for istio upgrades. 
   ##Setting releaseMirror.enabled: true will cause istio to use bundled in images from rancher/istio-installer to perfom an upgrade - this is ideal
   ##for airgap setups. Setting releaseMirror.enabled to false means istio will call externally to github to fetch the required assets.
   releaseMirror:
     enabled: false
-  ##A flag to run a sleep command `sleep <secondsSleep>s` to allow time to exec into istio-installer pod for debugging
+  ##Set the scondsSleep to run a sleep command `sleep <secondsSleep>s` to allow time to exec into istio-installer pod for debugging
   debug:
-    enabled: false
-    secondsSleep: 500
+    secondsSleep: 0
```

Comment:
Why did we drop the default from 500 to 0?

Reply:
I had a default value when we were using a separate flag for enabled/disabled debug mode, now that I removed the flag, any amount of seconds will cause the istio-installer job to sleep for x seconds.

Reply:
So I think we should avoid trying to make a breaking change to the Helm chart's behavior / defaults if possible.

Instead of deprecating the `installer.debug` Helm value, could we do something like this?

```yaml
{{- if .Values.installer.debug.enabled }}
- name: SECONDS_SLEEP
   value: {{ .Values.installer.debug.secondsSleep | quote }}
{{- end }}
```
This way, we can also keep the default secondsSleep at 500 but still remove passing in the unnecessary DEBUG env var.

Reply:
This is all brand new for 2.6.1/2.5.10. Won't be a breaking change. 
We've gone back and forth between two vars or one, me and Nick liked a single var. It doesn't matter so much as long as it works somehow.

Reply:
[100.0.0+up1.10.4](https://github.com/rancher/charts/tree/release-v2.6/charts/rancher-istio/rancher-istio) is the current released version of Istio, so Caleb is right, the debug flag does not exist in production at this time so it is not a breaking change to make adjustments to how it works. My vote is to leave it as a single var.

Reply:
sounds good to me 👍